src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq.c

```diff
@@ -2924,6 +2978,14 @@ static int xgq_vmr_probe(struct platform_device *pdev)
 		XGQ_WARN(xgq, "cannot load system dtb from shell, ret: %d", ret);
 		ret = 0;
 	}
+
+	ret = xgq_log_page_plm_sync(pdev, &xgq->xgq_vmr_plm_sync,
```

Comment:
This is wrong location, we should not query the plm only once when driver probe.
We actually should query it from the sysfs node read.

This is the difference between system_dtb and plm_log. 

I think even the system_dtb should not be in the probe. I will fix this issue soon.

Reply:
Overall, seems ok to me. You can leave the code as is. I will refactor this and system_dtb together.